paywall/src/_server.js

```diff
@@ -19,7 +19,8 @@ function _server(port, dev) {
 
           // assigning `query` into the params means that we still
           // get the query string passed to our application
-          if (pathname.match('/0x')) {
+          const fixedPathname = pathname.replace(/\/+/g, '/')
+          if (fixedPathname.match('/0x')) {
```

Comment:
`'https://staging.unlock-protocol.com//0x18326419236491649813649817634'.match('/0x')` returns a match for me without the preprocessing step, and `fixedPathname` doesn't seem to be used anywhere but this condition.